Sources/SwiftKuery/Insert.swift

```diff
@@ -36,14 +36,17 @@ public struct Insert: Query {
     /// An array of `AuxiliaryTable` which will be used in a query with a WITH clause.
     public private (set) var with: [AuxiliaryTable]?
     
+    /// A flag for return ID
+    public private (set) var returnID: Bool?
```

Comment:
I don't understand why this `Bool` is optional, aren't there only two options (true or false)?

Reply:
Yes, will change that